Hmm, well I reran `nixpkgs-review` locally, and it complained when building `lomiri.lomiri-api`:
```
In file included from /build/source/include/lomiri/util/GlibMemory.h:28,
                 from /build/source/test/gtest/lomiri/util/GlibMemory/GlibMemory_test.cpp:19:
/build/source/include/lomiri/util/ResourcePtr.h: In member function 'R lomiri::util::ResourcePtr<R, D>::release()':
/build/source/include/lomiri/util/ResourcePtr.h:390:20: error: 'logic_error' is not a member of 'std'
  390 |         throw std::logic_error("release() called on ResourcePtr without resource");
      |                    ^~~~~~~~~~~
/build/source/include/lomiri/util/ResourcePtr.h:23:1: note: 'std::logic_error' is defined in header '<stdexcept>'; did you forget to '#include <stdexcept>'?
   22 | #include <mutex>
  +++ |+#include <stdexcept>
   23 | #include <type_traits>
```

So you can save yourself a `nixpkgs-review`, the PR is borked as-is until `lomiri.lomiri-api` is fixed. :sweat_smile:

--- a/pkgs/development/libraries/libayatana-common/default.nix
+++ b/pkgs/development/libraries/libayatana-common/default.nix
@@ -4,11 +4,11 @@
 , gitUpdater
 , testers
 , cmake
-, cmake-extras
 , glib
 , gobject-introspection
 , gtest
 , intltool
+, lomiri
 , pkg-config
 , systemd
 , vala
@@ -28,7 +28,7 @@ stdenv.mkDerivation (finalAttrs: {
   postPatch = ''
     # Queries via pkg_get_variable, can't override prefix
     substituteInPlace data/CMakeLists.txt \
-      --replace 'DESTINATION "''${SYSTEMD_USER_UNIT_DIR}"' 'DESTINATION "${placeholder "out"}/lib/systemd/user"'
+      --replace 'pkg_get_variable(SYSTEMD_USER_UNIT_DIR systemd systemd_user_unit_dir)' 'set(SYSTEMD_USER_UNIT_DIR ''${CMAKE_INSTALL_PREFIX}/lib/systemd/user)'
   '';
 
   strictDeps = true;
@@ -42,8 +42,9 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   buildInputs = [
-    cmake-extras
+    lomiri.cmake-extras
     glib
+    lomiri.lomiri-url-dispatcher
     systemd
   ];
 
@@ -53,7 +54,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   cmakeFlags = [
     "-DENABLE_TESTS=${lib.boolToString finalAttrs.finalPackage.doCheck}"
-    "-DENABLE_LOMIRI_FEATURES=OFF"
+    "-DENABLE_LOMIRI_FEATURES=ON"
     "-DGSETTINGS_LOCALINSTALL=ON"
     "-DGSETTINGS_COMPILE=ON"
   ];

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -22437,9 +22437,7 @@ with pkgs;
 
   libavif = callPackage ../development/libraries/libavif { };
 
-  libayatana-common = callPackage ../development/libraries/libayatana-common {
-    inherit (lomiri) cmake-extras;
-  };
+  libayatana-common = callPackage ../development/libraries/libayatana-common { };
 
   libb2 = callPackage ../development/libraries/libb2 { };
 

